### PR TITLE
feat(PPDSC-2072): enable applitools logs

### PR DESCRIPTION
**Pull request by tbradbury**
PPDSC-2072

**What**

To enable quicker debugging of issue in Applitools this enables the log flag so we get extra information if the CircleCi console. This can then be downloaded and sent to Applitools.
2. Enables the Applitools log flag. I also update the Applitools packages
3. Extra log information inside the CicrleCi console for the jobs.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2072-enable-logs 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2072-enable-logs/storybook

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "dev": "concurrently --names \"src,storybook,docs\" \"yarn build:ts --watch\" \"yarn dev:storybook\" \"next site -p 8081\"",
     "dev:docs": "yarn build:icons:dev && concurrently --names \"src,docs\" \"yarn build:ts --watch\" \"next site -p 8081\"",
     "dev:storybook": "yarn build:icons:dev && start-storybook --ci -p 6006 -s ./fonts,./static",
-    "eyes:storybook": "eyes-storybook --conf applitools.components.config.js",
+    "eyes:storybook": "APPLITOOLS_SHOW_LOGS=true eyes-storybook --conf applitools.components.config.js",
```
Comment (Xin00163) [inline package.json:21]: Why not set in the config? Does this override the config setting?

Comment (tbradbury) [reply package.json:21]: If you add it in the config the Batched get split up

Comment (Xin00163) [reply package.json:21]: Interesting

**Review by Xin00163 — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**